### ISSUE TITLE
[super] Information Architecture

### ISSUE THREAD

**Issue by rufuspollock** [okfn/handbook#94]
The overall structure of the site:

```
# home page setting out overview and content
/

# high level info on how to contribute to labs
/contribute/

# general intro to data and data wrangling (+ projects)
/data/

# data patterns
# maybe move to /data/patterns/ ?? or even just move patterns under /data/
/patterns/

# core datasets project
/core-datasets/
```

**Comment by rufuspollock** [comment]
@gsilvapt i have started this issue so we can discuss site layout (and priorities)

**Comment by gsilvapt** [comment]
@rgrp , thanks, indeed super useful!

Even though I have been reading it for a couple of days now, I am not sure what you guys mean with "Data Patterns". It previously was a bunch of Data Wrangling tutorials, in which I haven't done any change. 

As for the Core Datasets projects, I wrote some tutorials and guides. They seem repeated in the FrictionlessData's tutorials too. After checking Dan's comment in another issue (https://github.com/frictionlessdata/ideas/issues/240#issuecomment-217420921), I am not sure if I am understanding the scope of both websites.

Nevertheless, until we decide on which guides are going where, I can definitely work on something under `home page` and `/contribute/`.

**Comment by rufuspollock** [comment]
@gsilvapt i agree that "data patterns" were basically recipes for working with data. The "patterns" reference is to "software patterns". Right now let us not worry too much on that front.

**Comment by gsilvapt** [comment]
Okay, sure. Can we discuss then the scope of **this** handbook? Like its usage, types of content you're expecting, target, etc etc?

**Comment by rufuspollock** [comment]
@gsilvapt **great** questions. Can you open a new issue called "Audience and User Stories". I also recommend reading: https://github.com/rgrp/project-management/blob/master/user-stories.md

**Comment by gsilvapt** [comment]
@rgrp Done in issue #97.
